target/linux/pistachio/base-files/lib/netifd/proto/lowpan.sh

```diff
@@ -23,20 +24,18 @@ proto_lowpan_setup() {
         proto_init_update "$ifname" 1
 
         # Wifi mac address must be part of DTB to compute 6Lowpan MAC
-        if [ ! -e /proc/device-tree/uccp@18480000/mac-address0 ]; then
-                logger "lowpan: Could not find wifi STA MAC address"
-        else
-                local WIFI_STA_MAC=0x$(hexdump -n 6 -v -e '/1 "%02X"' /proc/device-tree/uccp@18480000/mac-address0)
-
-                # 6LOWPAN_MAC = WIFI_STA_MAC + 5
-                local LOWPAN_OTP_OFFSET=0x5
-                local LOWPAN_MAC=$(printf "%012X" $((WIFI_STA_MAC + LOWPAN_OTP_OFFSET)))
-                LOWPAN_MAC=$(macaddr_setbit_la $(macaddr_canonicalize $LOWPAN_MAC))
-
-                # Compute EUI from MAC
-                local WPAN_EUI=$(echo $LOWPAN_MAC | cut -d: -f1,2,3):ff:fe:$(echo $LOWPAN_MAC | cut -d: -f4,5,6)
-                ip link set $interface address $WPAN_EUI
-        fi
+        local PRIMARY_MAC_ADDR=$(get_primary_macaddr)
```

Comment:
all this business logic of obtaining lowpan mac address from wifi mac (or primary mac) address should be in mac.sh.
Here i was expecting to see just get_lowpan_mac call. and setting up lowpan interface with that mac